internal/server/routes.go

```diff
@@ -157,33 +155,85 @@ func add[Req, Res any](a *API, r *gin.RouterGroup, route route[Req, Res]) {
 		a.register(openAPIRouteDefinition(route))
 	}
 
-	handlers := includeRewritesFor(a, route.method, route.path)
-	handlers = append(handlers, func(c *gin.Context) {
+	handler := func(c *gin.Context) {
```

Comment:
This is the important part of the change. These 4 functions (below) and 2 types handle the version header, and allow us to replace all of apimigrator.